### ISSUE TITLE
setting defaultTypes.json as JSON schema causes TypeError

### ISSUE THREAD

**Issue by mikesbrown** [brimsec/brim#792]
When I follow instructions at https://github.com/brimsec/brim/wiki/Zeek-JSON-Import to set a JSON schema, the following is logged to developer tools console:
```
TypeError: Invalid attempt to destructure non-iterable instance
    at _nonIterableRest (/Applications/Brim.app/Contents/Resources/app/dist/js/brim/form.js:10)
    at _slicedToArray (/Applications/Brim.app/Contents/Resources/app/dist/js/brim/form.js:8)
    at _callee$ (/Applications/Brim.app/Contents/Resources/app/dist/js/brim/form.js:57)
    at tryCatch (/Applications/Brim.app/Contents/Resources/app/node_modules/regenerator-runtime/runtime.js:45)
    at Generator.invoke [as _invoke] (/Applications/Brim.app/Contents/Resources/app/node_modules/regenerator-runtime/runtime.js:271)
    at Generator.prototype.<computed> [as next] (/Applications/Brim.app/Contents/Resources/app/node_modules/regenerator-runtime/runtime.js:97)
    at asyncGeneratorStep (/Applications/Brim.app/Contents/Resources/app/dist/js/brim/form.js:16)
    at _next (/Applications/Brim.app/Contents/Resources/app/dist/js/brim/form.js:18)
```
Note that I tried this on an installed release package (`node scripts/release --darwin`) to work around #791 .

The file I wanted to upload to this is `config/defaultTypes.json`. Commit hash is 416f14a52bc2da9e423e6fac202bf0a4fd879cee.

**Comment by mikesbrown** [comment]
This blocks #788. I highly suggest whoever fixes this take on #788 as part of the work to help prevent future regressions.

**Comment by philrz** [comment]
Verified in Brim commit `ff40f0e`. In the attached video I successfully pick the `config/defaultTypes.json` as my typing schema, it's accepted without error, and I'm able to use it for NDJSON import.

[Verify.zip](https://github.com/brimsec/brim/files/4689625/Verify.zip)

Thanks @jameskerr!
